### PR TITLE
Resolved task #499

### PR DESCRIPTION
**Pull request by Beniamin-Dolega**
Renamed SyncToRemoteStatus ==> Relation to Remote
Renamed Relation ==> SyncToRemoteStatus aditionally making it a top-level enum

### CODE REVIEW

File: backendApi/src/main/java/com/virtuslab/gitmachete/backend/api/RelationToRemote.java [hunk 1]
```diff
@@ -0,0 +1,22 @@
+package com.virtuslab.gitmachete.backend.api;
+
+import lombok.Data;
+import org.checkerframework.checker.nullness.qual.Nullable;
+
+@Data(staticConstructor = "of")
+// So that Interning Checker doesn't complain about enum comparison (by `equals` and not by `==`) in Lombok-generated `equals`
+@SuppressWarnings("interning:unnecessary.equals")
+public class RelationToRemote {
+  // TODO (#499): Resolve "status" (to parent) and "relation" (to remote) inconsistency
```
Comment (PawelLipski) [inline backendApi/src/main/java/com/virtuslab/gitmachete/backend/api/RelationToRemote.java:10]: Remove this `TODO` as this issue will be resolved when this PR gets merged

File: backendImpl/src/main/java/com/virtuslab/gitmachete/backend/impl/GitMacheteRepository.java [hunk 2]
```diff
@@ -638,56 +638,56 @@ IGitMacheteRepositorySnapshot createSnapshot(IBranchLayout branchLayout) throws
     }
 
     @UIThreadUnsafe
-    private SyncToRemoteStatus deriveSyncToRemoteStatus(IGitCoreLocalBranchSnapshot coreLocalBranch) throws GitCoreException {
+    private RelationToRemote deriveSyncToRemoteStatus(IGitCoreLocalBranchSnapshot coreLocalBranch) throws GitCoreException {
```
Comment (PawelLipski) [inline backendImpl/src/main/java/com/virtuslab/gitmachete/backend/impl/GitMacheteRepository.java:641]: Rename the method adequately

File: backendImpl/src/test/java/com/virtuslab/gitmachete/backend/integration/ParentInferenceIntegrationTestSuite.java [hunk 3]
```diff
@@ -24,6 +25,7 @@
 import com.virtuslab.gitmachete.backend.impl.GitMacheteRepositoryCache;
 import com.virtuslab.gitmachete.testcommon.BaseGitRepositoryBackedIntegrationTestSuite;
 
+@Ignore
```
Comment (PawelLipski) [inline backendImpl/src/test/java/com/virtuslab/gitmachete/backend/integration/ParentInferenceIntegrationTestSuite.java:28]: Pls add a `TODO (#753): un-ignore this suite`

Comment (PawelLipski) [reply backendImpl/src/test/java/com/virtuslab/gitmachete/backend/integration/ParentInferenceIntegrationTestSuite.java:28]: Your next task :)

File: backendImpl/src/test/java/com/virtuslab/gitmachete/backend/integration/StatusAndDiscoverIntegrationTestSuite.java [hunk 4]
```diff
@@ -30,17 +31,11 @@
...
-import com.virtuslab.gitmachete.backend.api.IManagedBranchSnapshot;
-import com.virtuslab.gitmachete.backend.api.INonRootManagedBranchSnapshot;
-import com.virtuslab.gitmachete.backend.api.IRootManagedBranchSnapshot;
-import com.virtuslab.gitmachete.backend.api.SyncToParentStatus;
-import com.virtuslab.gitmachete.backend.api.SyncToRemoteStatus;
+import com.virtuslab.gitmachete.backend.api.*;
 import com.virtuslab.gitmachete.backend.impl.GitMacheteRepositoryCache;
 import com.virtuslab.gitmachete.testcommon.BaseGitRepositoryBackedIntegrationTestSuite;
 
+@Ignore
```
Comment (PawelLipski) [inline backendImpl/src/test/java/com/virtuslab/gitmachete/backend/integration/StatusAndDiscoverIntegrationTestSuite.java:38]: Ditto, pls add a TODO

File: backendImpl/src/test/java/com/virtuslab/gitmachete/backend/integration/StatusAndDiscoverIntegrationTestSuite.java [hunk 5]
```diff
@@ -214,13 +209,13 @@ private void printCommonParts(IManagedBranchSnapshot branch, List<INonRootManage
       sb.append("  ");
       sb.append(customAnnotation.get());
     }
-    val syncToRemote = branch.getSyncToRemoteStatus();
+    val syncToRemote = branch.getRelationToRemote();
```
Comment (PawelLipski) [inline backendImpl/src/test/java/com/virtuslab/gitmachete/backend/integration/StatusAndDiscoverIntegrationTestSuite.java:212]: Rename the local variable

Comment (PawelLipski) [reply backendImpl/src/test/java/com/virtuslab/gitmachete/backend/integration/StatusAndDiscoverIntegrationTestSuite.java:212]: Btw, `val` (short for _value_) means two things:
1. the type is automatically guessed based on what you assign to the variable (you don't need to specify it yourself)
2. the variable is **final** (can't be ever reassigned), so it's not really a variable but right more of a constant

File: backendImpl/src/test/java/com/virtuslab/gitmachete/backend/unit/GitMacheteRepository_deriveSyncToRemoteStatusToRemoteUnitTestSuite.java [hunk 6]
```diff
@@ -16,9 +16,10 @@
 import com.virtuslab.gitcore.api.IGitCoreCommit;
 import com.virtuslab.gitcore.api.IGitCoreLocalBranchSnapshot;
 import com.virtuslab.gitcore.api.IGitCoreRemoteBranchSnapshot;
+import com.virtuslab.gitmachete.backend.api.RelationToRemote;
 import com.virtuslab.gitmachete.backend.api.SyncToRemoteStatus;
 
-public class GitMacheteRepository_deriveSyncToRemoteStatusUnitTestSuite extends BaseGitMacheteRepositoryUnitTestSuite {
+public class GitMacheteRepository_deriveSyncToRemoteStatusToRemoteUnitTestSuite extends BaseGitMacheteRepositoryUnitTestSuite {
```
Comment (PawelLipski) [inline backendImpl/src/test/java/com/virtuslab/gitmachete/backend/unit/GitMacheteRepository_deriveSyncToRemoteStatusToRemoteUnitTestSuite.java:22]: Whoops too much stuff in this name

File: backendImpl/src/test/java/com/virtuslab/gitmachete/backend/unit/GitMacheteRepository_deriveSyncToRemoteStatusToRemoteUnitTestSuite.java [hunk 7]
```diff
@@ -28,7 +29,7 @@
   private final IGitCoreCommit coreRemoteBranchCommit = PowerMockito.mock(IGitCoreCommit.class);
 
   @SneakyThrows
-  private SyncToRemoteStatus invokeDeriveSyncToRemoteStatus(IGitCoreLocalBranchSnapshot coreLocalBranch) {
+  private RelationToRemote invokeDeriveSyncToRemoteStatus(IGitCoreLocalBranchSnapshot coreLocalBranch) {
     return Whitebox.invokeMethod(aux(), "deriveSyncToRemoteStatus", coreLocalBranch);
```
Comment (PawelLipski) [inline backendImpl/src/test/java/com/virtuslab/gitmachete/backend/unit/GitMacheteRepository_deriveSyncToRemoteStatusToRemoteUnitTestSuite.java:33]: Hmm is this method still called `deriveSyncToRemoteStatus` even though it returns a `RelationToRemote`?

Comment (PawelLipski) [reply backendImpl/src/test/java/com/virtuslab/gitmachete/backend/unit/GitMacheteRepository_deriveSyncToRemoteStatusToRemoteUnitTestSuite.java:33]: Tip: Ctrl+Shift+Alt+N to locate any **symbol**, including class, method etc.

Comment (PawelLipski) [reply backendImpl/src/test/java/com/virtuslab/gitmachete/backend/unit/GitMacheteRepository_deriveSyncToRemoteStatusToRemoteUnitTestSuite.java:33]: (while Ctrl+N just looks for any **class**)

File: docs/features.md [hunk 8]
```diff
@@ -10,7 +10,7 @@ You can also use `Ctrl + Alt + Shift + M` shortcut to open it.
 
 ## Branch layout graph
 
-For each branch, Git Machete indicates the relation to each of its child branches.
+For each branch, Git Machete indicates the syncToRemoteStatus to each of its child branches.
```
Comment (PawelLipski) [inline docs/features.md:13]: Whoops

File: frontendActions/src/main/java/com/virtuslab/gitmachete/frontend/actions/base/BasePushBranchAction.java [hunk 9]
```diff
@@ -47,7 +47,7 @@ public IEnhancedLambdaLogger log() {
   }
 
   @Override
-  public List<SyncToRemoteStatus.Relation> getEligibleRelations() {
+  public List<SyncToRemoteStatus> getEligibleRelations() {
```
Comment (PawelLipski) [inline frontendActions/src/main/java/com/virtuslab/gitmachete/frontend/actions/base/BasePushBranchAction.java:50]: Rename the method adequately

File: frontendResourceBundles/src/main/resources/GitMacheteBundle.properties [hunk 10]
```diff
@@ -160,7 +160,7 @@ action.GitMachete.ISyncToRemoteStatusDependentAction.description.sync-to-remote-
 action.GitMachete.ISyncToRemoteStatusDependentAction.description.sync-to-remote-status.relation.diverged-from.and-older-than-remote=diverged from (& older than) its remote
 action.GitMachete.ISyncToRemoteStatusDependentAction.description.sync-to-remote-status.relation.in-sync-to-remote=in sync with its remote
 action.GitMachete.ISyncToRemoteStatusDependentAction.description.sync-to-remote-status.relation.untracked=untracked
-action.GitMachete.ISyncToRemoteStatusDependentAction.description.sync-to-remote-status.relation.unknown=in unknown relation ''{0}'' to its remote
+action.GitMachete.ISyncToRemoteStatusDependentAction.description.sync-to-remote-status.relation.unknown=in unknown syncToRemoteStatus ''{0}'' to its remote
```
Comment (PawelLipski) [inline frontendResourceBundles/src/main/resources/GitMacheteBundle.properties:163]: Let's leave `relation` here, as it was (that's just a message for a user, we can be a little bit informal here, as compared to the naming we use in the code itself)

File: frontendUiTableImpl/src/main/java/com/virtuslab/gitmachete/frontend/ui/impl/table/DemoGitMacheteRepositorySnapshot.java [hunk 11]
```diff
@@ -88,8 +89,8 @@ public DemoGitMacheteRepositorySnapshot() {
     this.roots = List.of(root);
   }
 
-  static SyncToRemoteStatus getSTRSofRelation(SyncToRemoteStatus.Relation relation) {
-    return SyncToRemoteStatus.of(relation, "origin");
+  static RelationToRemote getSTRSofRelation(SyncToRemoteStatus syncToRemoteStatus) {
```
Comment (PawelLipski) [inline frontendUiTableImpl/src/main/java/com/virtuslab/gitmachete/frontend/ui/impl/table/DemoGitMacheteRepositorySnapshot.java:92]: Rename the method, you're now getting _relation of status_ after the renames, not _status of relation_ ;)

File: frontendUiTableImpl/src/main/java/com/virtuslab/gitmachete/frontend/ui/impl/cell/BranchOrCommitCellRendererComponent.java [hunk 12]
```diff
@@ -251,17 +251,17 @@ private static BufferedImage getGraphImage(JTable table, @NonNegative int maxGra
...
-    return Match(relation.getRelation()).of(
+  private static JBColor getColor(RelationToRemote relation) {
+    return Match(relation.getSyncToRemoteStatus()).of(
         Case($(isIn(NoRemotes, InSyncToRemote)), TRANSPARENT),
         Case($(Untracked), ORANGE),
         Case($(isIn(AheadOfRemote, BehindRemote, DivergedFromAndNewerThanRemote, DivergedFromAndOlderThanRemote)), RED));
   }
 
-  private static String getSyncToRemoteStatusBasedLabel(SyncToRemoteStatus status) {
+  private static String getSyncToRemoteStatusBasedLabel(RelationToRemote status) {
```
Comment (PawelLipski) [inline frontendUiTableImpl/src/main/java/com/virtuslab/gitmachete/frontend/ui/impl/cell/BranchOrCommitCellRendererComponent.java:261]: Rename the param

File: frontendActions/src/main/java/com/virtuslab/gitmachete/frontend/actions/base/ISyncToRemoteStatusDependentAction.java [hunk 13]
```diff
@@ -64,9 +64,9 @@ default void syncToRemoteStatusDependentActionUpdate(AnActionEvent anActionEvent
               getActionNameForDescription(), getQuotedStringOrCurrent(branchName)));
       return;
     }
-    val syncToRemoteStatus = gitMacheteBranch.getSyncToRemoteStatus();
+    val syncToRemoteStatus = gitMacheteBranch.getRelationToRemote();
 
-    SyncToRemoteStatus.Relation relation = syncToRemoteStatus.getRelation();
+    SyncToRemoteStatus relation = syncToRemoteStatus.getSyncToRemoteStatus();
```
Comment (PawelLipski) [inline frontendActions/src/main/java/com/virtuslab/gitmachete/frontend/actions/base/ISyncToRemoteStatusDependentAction.java:69]: Rename the var

File: frontendActions/src/main/java/com/virtuslab/gitmachete/frontend/actions/base/ISyncToRemoteStatusDependentAction.java [hunk 14]
```diff
@@ -64,9 +64,9 @@ default void syncToRemoteStatusDependentActionUpdate(AnActionEvent anActionEvent
               getActionNameForDescription(), getQuotedStringOrCurrent(branchName)));
       return;
     }
-    val syncToRemoteStatus = gitMacheteBranch.getSyncToRemoteStatus();
+    val syncToRemoteStatus = gitMacheteBranch.getRelationToRemote();
```
Comment (PawelLipski) [inline frontendActions/src/main/java/com/virtuslab/gitmachete/frontend/actions/base/ISyncToRemoteStatusDependentAction.java:67]: Rename the val

File: frontendActions/src/main/java/com/virtuslab/gitmachete/frontend/actions/base/ISyncToRemoteStatusDependentAction.java [hunk 15]
```diff
@@ -42,7 +42,7 @@ default String getEnabledDescriptionFormat() {
    *
    * @return a list of relations for which the action should be enabled (not necessarily visible)
    */
-  List<SyncToRemoteStatus.Relation> getEligibleRelations();
+  List<SyncToRemoteStatus> getEligibleRelations();
```
Comment (PawelLipski) [inline frontendActions/src/main/java/com/virtuslab/gitmachete/frontend/actions/base/ISyncToRemoteStatusDependentAction.java:45]: As mentioned, rename this method

File: frontendActions/src/main/java/com/virtuslab/gitmachete/frontend/actions/base/ISyncToRemoteStatusDependentAction.java [hunk 16]
```diff
@@ -33,7 +33,7 @@ default String getEnabledDescriptionFormat() {
   }
 
   /**
-   * This method provides a list of {@link SyncToRemoteStatus.Relation}s for which the action should be ENABLED.
+   * This method provides a list of {@link SyncToRemoteStatus}s for which the action should be ENABLED.
```
Comment (PawelLipski) [inline frontendActions/src/main/java/com/virtuslab/gitmachete/frontend/actions/base/ISyncToRemoteStatusDependentAction.java:36]: Nitpick:
```suggestion
   * This method provides a list of {@link SyncToRemoteStatus}es for which the action should be ENABLED.
```

File: backendImpl/src/test/java/com/virtuslab/gitmachete/backend/unit/GitMacheteRepository_deriveSyncToRemoteStatusToRemoteUnitTestSuite.java (hunk 6 above)
Comment (Beniamin-Dolega) [reply backendImpl/src/test/java/com/virtuslab/gitmachete/backend/unit/GitMacheteRepository_deriveSyncToRemoteStatusToRemoteUnitTestSuite.java:22]: Should it be called `GitMacheteRepository_deriveSyncToRemoteStatusUnitTestSuite` or `GitMacheteRepository_deriveRelationToRemoteUnitTestSuite` now?

File: frontendActions/src/main/java/com/virtuslab/gitmachete/frontend/actions/base/BasePushBranchAction.java (hunk 9 above)
Comment (Beniamin-Dolega) [reply frontendActions/src/main/java/com/virtuslab/gitmachete/frontend/actions/base/BasePushBranchAction.java:50]: Should it be named `getEligibleStatuses` now?

File: backendImpl/src/main/java/com/virtuslab/gitmachete/backend/impl/GitMacheteRepository.java [hunk 17]
```diff
@@ -401,7 +401,7 @@ IGitMacheteRepositorySnapshot createSnapshot(IBranchLayout branchLayout) throws
       IGitCoreCommit corePointedCommit = coreLocalBranch.getPointedCommit();
 
       val pointedCommit = new CommitOfManagedBranch(corePointedCommit);
-      val syncToRemoteStatus = deriveSyncToRemoteStatus(coreLocalBranch);
+      val syncToRemoteStatus = deriveRelationToRemote(coreLocalBranch);
```
Comment (PawelLipski) [inline backendImpl/src/main/java/com/virtuslab/gitmachete/backend/impl/GitMacheteRepository.java:404]: Rename the val

File: backendImpl/src/main/java/com/virtuslab/gitmachete/backend/impl/GitMacheteRepository.java [hunk 18]
```diff
@@ -457,7 +457,7 @@ IGitMacheteRepositorySnapshot createSnapshot(IBranchLayout branchLayout) throws
       }
 
       val pointedCommit = new CommitOfManagedBranch(corePointedCommit);
-      val syncToRemoteStatus = deriveSyncToRemoteStatus(coreLocalBranch);
+      val syncToRemoteStatus = deriveRelationToRemote(coreLocalBranch);
```
Comment (PawelLipski) [inline backendImpl/src/main/java/com/virtuslab/gitmachete/backend/impl/GitMacheteRepository.java:460]: Ditto

File: frontendActions/src/main/java/com/virtuslab/gitmachete/frontend/actions/base/BasePushBranchAction.java (hunk 9 above)
Comment (PawelLipski) [reply frontendActions/src/main/java/com/virtuslab/gitmachete/frontend/actions/base/BasePushBranchAction.java]: Yes (pls don't resolve a thread if you wait for a response coz the reviewers are then more likely to overlook)

File: backendImpl/src/test/java/com/virtuslab/gitmachete/backend/unit/GitMacheteRepository_deriveSyncToRemoteStatusToRemoteUnitTestSuite.java (hunk 6 above)
Comment (PawelLipski) [reply backendImpl/src/test/java/com/virtuslab/gitmachete/backend/unit/GitMacheteRepository_deriveSyncToRemoteStatusToRemoteUnitTestSuite.java]: `GitMacheteRepository_deriveRelationToRemoteUnitTestSuite` (current name is okay)

File: docs/features.md [hunk 19]
```diff
@@ -188,7 +188,7 @@ master
 Two of them, `allow-ownership-link` and `call-ws`, have a custom annotation &mdash;
 it's an arbitrary description displayed next to the given branch (in this case, pull request numbers).
 
-The relation between these branches is determined by indentations &mdash; here single indent is 4 spaces, but a tab can be used as well.
+The syncToRemoteStatus between these branches is determined by indentations &mdash; here single indent is 4 spaces, but a tab can be used as well.
```
Comment (PawelLipski) [inline docs/features.md:191]: Revert

File: docs/features.md [hunk 20]
```diff
@@ -217,7 +217,7 @@ On the left side bar you can find other actions (from top to bottom):
     - ![](left_bar_actions/rebase.png) **Rebase Current Branch Onto Parent**
     - ![](left_bar_actions/override_forkpoint.png) **Override Fork Point of Current Branch**
 
-  Available action is selected based on a relation between a current branch and its parent and remote branch.
+  Available action is selected based on a syncToRemoteStatus between a current branch and its parent and remote branch.
```
Comment (PawelLipski) [inline docs/features.md:220]: Revert

File: frontendActions/src/main/java/com/virtuslab/gitmachete/frontend/actions/base/ISyncToRemoteStatusDependentAction.java [hunk 21]
```diff
@@ -64,10 +64,10 @@ default void syncToRemoteStatusDependentActionUpdate(AnActionEvent anActionEvent
               getActionNameForDescription(), getQuotedStringOrCurrent(branchName)));
       return;
     }
-    val syncToRemoteStatus = gitMacheteBranch.getSyncToRemoteStatus();
+    val RelationToRemote = gitMacheteBranch.getRelationToRemote();
```
Comment (PawelLipski) [inline frontendActions/src/main/java/com/virtuslab/gitmachete/frontend/actions/base/ISyncToRemoteStatusDependentAction.java:67]: Lower case, as it's a local val, not e.g. a class

File: frontendActions/src/main/java/com/virtuslab/gitmachete/frontend/actions/base/ISyncToRemoteStatusDependentAction.java [hunk 22]
```diff
@@ -64,10 +64,10 @@ default void syncToRemoteStatusDependentActionUpdate(AnActionEvent anActionEvent
...
 
-    SyncToRemoteStatus.Relation relation = syncToRemoteStatus.getRelation();
-    val isRelationEligible = getEligibleRelations().contains(relation);
+    SyncToRemoteStatus status = RelationToRemote.getSyncToRemoteStatus();
+    val isRelationEligible = getEligibleStatuses().contains(status);
```
Comment (PawelLipski) [inline frontendActions/src/main/java/com/virtuslab/gitmachete/frontend/actions/base/ISyncToRemoteStatusDependentAction.java:70]:
```suggestion
    val isStatusEligible = getEligibleStatuses().contains(status);
```

File: frontendActions/src/main/java/com/virtuslab/gitmachete/frontend/actions/base/ISyncToRemoteStatusDependentAction.java [hunk 23]
```diff
@@ -78,21 +78,21 @@ default void syncToRemoteStatusDependentActionUpdate(AnActionEvent anActionEvent
       presentation.setEnabled(false);
 
       // @formatter:off
-      val desc = Match(relation).of(
-          Case($(SyncToRemoteStatus.Relation.AheadOfRemote),
+      val desc = Match(status).of(
+          Case($(SyncToRemoteStatus.AheadOfRemote),
               getString("action.GitMachete.ISyncToRemoteStatusDependentAction.description.sync-to-remote-status.relation.ahead-of-remote")),
```
Comment (PawelLipski) [inline frontendActions/src/main/java/com/virtuslab/gitmachete/frontend/actions/base/ISyncToRemoteStatusDependentAction.java:83]: TBH I'd remove the `.relation` part from all the `action.GitMachete.ISyncToRemoteStatusDependentAction.` property keys

Comment (PawelLipski) [reply frontendActions/src/main/java/com/virtuslab/gitmachete/frontend/actions/base/ISyncToRemoteStatusDependentAction.java:83]: You need to do the change both where the keys are referenced (so, in this Java file) and where they're declared (i.e. in `frontendResourceBundles/src/main/resources/GitMacheteBundle.properties`)

File: frontendUiTableImpl/src/main/java/com/virtuslab/gitmachete/frontend/ui/impl/cell/BranchOrCommitCellRendererComponent.java [hunk 24]
```diff
@@ -173,9 +173,9 @@ public BranchOrCommitCellRendererComponent(
         append(CELL_TEXT_FRAGMENTS_SPACING + statusHookOutput.get(), GRAY_ATTRIBUTES);
       }
 
-      SyncToRemoteStatus syncToRemoteStatus = branchItem.getSyncToRemoteStatus();
-      val textAttributes = new SimpleTextAttributes(STYLE_PLAIN, getColor(syncToRemoteStatus));
-      String remoteStatusLabel = getSyncToRemoteStatusBasedLabel(syncToRemoteStatus);
+      RelationToRemote relationToRemote = branchItem.getRelationToRemote();
+      val textAttributes = new SimpleTextAttributes(STYLE_PLAIN, getColor(relationToRemote));
+      String remoteStatusLabel = getRelationToRemoteBasedLabel(relationToRemote);
```
Comment (PawelLipski) [inline frontendUiTableImpl/src/main/java/com/virtuslab/gitmachete/frontend/ui/impl/cell/BranchOrCommitCellRendererComponent.java:178]: Nitpick:
```suggestion
      String relationToRemoteLabel = getRelationToRemoteBasedLabel(relationToRemote);
```

File: frontendUiTableImpl/src/main/java/com/virtuslab/gitmachete/frontend/ui/impl/cell/BranchOrCommitCellRendererComponent.java [hunk 25]
```diff
@@ -251,17 +251,17 @@ private static BufferedImage getGraphImage(JTable table, @NonNegative int maxGra
...
-    return Match(relation.getRelation()).of(
+  private static JBColor getColor(RelationToRemote relation) {
+    return Match(relation.getSyncToRemoteStatus()).of(
         Case($(isIn(NoRemotes, InSyncToRemote)), TRANSPARENT),
         Case($(Untracked), ORANGE),
         Case($(isIn(AheadOfRemote, BehindRemote, DivergedFromAndNewerThanRemote, DivergedFromAndOlderThanRemote)), RED));
   }
 
-  private static String getSyncToRemoteStatusBasedLabel(SyncToRemoteStatus status) {
+  private static String getRelationToRemoteBasedLabel(RelationToRemote status) {
```
Comment (PawelLipski) [inline frontendUiTableImpl/src/main/java/com/virtuslab/gitmachete/frontend/ui/impl/cell/BranchOrCommitCellRendererComponent.java:261]: Rename the param

**Review by PawelLipski — APPROVED** [review]
Good job 🎉 as soon as CircleCI passes, you can Squash & Merge this pull request via button here on GitHub

**Pull request merged**